### ISSUE TITLE
Adobe AIR detection

### ISSUE THREAD

**Issue by teuf22** [ua-parser/uap-core#190]
Adobe AIR detection is broken (detected as AppleMail) because AIR versions are X.Y and no longer X.Y.Z. Here are some recent examples:

```
# AIR on Samsung Galaxy Tab
"Mozilla/5.0 (Android; U; en-US) AppleWebKit/533.19.4 (KHTML, like Gecko) AdobeAIR/23.0"
Expected: AdobeAIR 23.0|Android
# AIR on mac desktop
"Mozilla/5.0 (Macintosh; U; Intel Mac OS X; en-US) AppleWebKit/533.19.4 (KHTML, like Gecko) AdobeAIR/19.0"
Expected: AdobeAIR 19.0|Mac OS X
# AIR on iPad
"Mozilla/5.0 (iOS; U; en-US) AppleWebKit/533.19.4 (KHTML, like Gecko) AdobeAIR/19.0"
Expected: AdobeAIR 19.0|iOS
```

**Comment by teuf22** [comment]
Thank you for the quick fix!

For the third example given (AIR on iPad), the OS detection has a problem, ua-parser gives 'AdobeAIR 19.0|Other', can this be corrected to show 'AdobeAIR 19.0|iOS' ?

**Comment by commenthol** [comment]
Reopened - fix required for OS
